docs/guide/docs/channels/web.md

```diff
@@ -46,19 +46,18 @@ How to set up the information page:
 
 You will see the page when starting a new conversation. The page is always accessible by clicking on the information icon in the top right corner of the chat window.
 
-> **\*\*** We edited the `global` configuration file for the sake of simplicity. To enable the bot information page on a single bot, you will need to copy the file `data/global/config/channel-web.json` to your bot folder `data/bots/BOT_NAME/config/channel-web.json` and edit that file.
+> **\*\*** We edited the `global` configuration file for the sake of simplicity. To enable the bot information page on a single bot, visit that bot's studio UI and select **config** (the last icon on the left-side menu with a cog).
 
 ## Show and hide automatically
-
-If the default Botpress button doesn't work for you, it can be changed by adding a `click` event listener to any element on the page. You will also need to pass the `hideWidget` key to your `init` function like this:
+If the default Botpress button doesn't work for you, pass the `hideWidget` key to your `init` function. By so doing, the default chat button will not show up on your website.
```

Comment:
By Botpress Button what do we mean exactly? The webchat icon found at the bottom right corner of a website? @Michael-N-M